2/contrib/jenkins/install-jenkins-core-plugins.sh

```diff
@@ -2,6 +2,8 @@
 
 set -o pipefail
 
+
+jenkins_version=$(cat /opt/openshift/jenkins-version.txt)
```

Comment:
add an echo that display the value of `jenkins_version`

Reply:
done

Reply:
ok now see

```
2022-02-02T16:54:45.838915482Z Jenkins version: 2.319.2
```

in the image build log.